requirements.txt

```diff
@@ -6,3 +6,4 @@ djangorestframework>=3.3,<3.12
 django-model-utils~=4.0.0
 netdiff~=0.9.0
 jsonfield>=3.1.0,<4.0.0
+drf-yasg~=1.17.1
```

Comment:
this dependency is included in openwisp-utils please depend on `openwisp-utils[rest]~=0.6.0`
similar to https://github.com/openwisp/openwisp-firmware-upgrader/blob/master/setup.py#L39